src/main/java/com/hazelcast/web/HazelcastHttpSession.java

```diff
@@ -48,6 +49,8 @@
 
     private final boolean stickySession;
     private final boolean deferredWrite;
+    // true when session is fetched from local cache
+    private boolean needNotify = true;
```

Comment:
Instead of `needNotify`, a more explanatory wording might be better. Such as `keepRemoteSessionActive`, or `keepRemoteActive`.

Reply:
I wanted to include `need` keyword in addition to your suggestion since we behave this as a flag at the end of a request.  Renamed as `needKeepRemoteActive`.